Development/nmos-cpp-node/node_implementation.cpp

```diff
@@ -418,6 +426,7 @@ void node_implementation_init(nmos::node_model& model, slog::base_gate& gate)
             if (impl::ports::video == port)
             {
                 receiver = nmos::make_video_receiver(receiver_id, device_id, nmos::transports::rtp, interface_names, model.settings);
+                receiver.data[U("caps")][U("media_types")][0] = value::string(nmos::media_types::video_H264.name);
```

Comment:
Although we can work with nmos-cpp as is, if we decide to push some of the `nmos::details` functions noted in _nmos/video_h264.cpp_ into _nmos/sdp_utils.h_, another thing to do would be to add an overload of `nmos::make_video_receiver` in _nmos/node_resources.h_ that takes a `media_type` or vector of `media_types`, similar to `nmos::make_coded_video_flow`. That would avoid this little dance.